modules/gapi/src/api/render_priv.hpp

```diff
@@ -24,6 +24,24 @@ GAPI_EXPORTS void BGR2NV12(const cv::Mat& bgr, cv::Mat& y_plane, cv::Mat& uv_pla
 
 } // namespace draw
 } // namespace wip
+
+namespace ocv
+{
+    void createBitMask(const wip::draw::Text& text_prim,
+                       cv::Mat& mask,
+                       int* baseline);
+
+    // FIXME only for tests
+    GAPI_EXPORTS void blendImage(const cv::Mat& img,
+            const cv::Mat& alpha,
```

Comment:
Indentation

Reply:
I don't see any tests on that. Please, remove from the header or test properly (I would recommend removing).

Reply:
Removed export